spec/forms/provider_interface/offer_wizard_spec.rb

```diff
@@ -173,26 +161,17 @@
         let(:current_step) { :study_modes }
 
         context 'when there are multiple locations available' do
-          before do
-            course_option = class_double(CourseOption, one?: false)
-
-            allow(CourseOption).to receive(:where).and_return(course_option)
-          end
-
           it 'returns :locations' do
+            allow(query_service).to receive(:available_course_options).and_return([create(:course_option), create(:course_option)])
```

Comment:
Can we keep the existing pattern of the spec, and set any expectations in the before block rather than the example, as it makes it easier to distinguish the setup and the expectation of each example.